### PR TITLE
Remove compose bom

### PR DESCRIPTION
**Pull request by yamal-alm**
### :goal_net: What's the goal?
Remove Compose BoM since it is right now giving us some dependency resolution problems at the time of importing Mistica in other projects (missing dependencies, not found classes after on release builds, etc).

### :construction: How do we do it?
Revert the commit 7c9b20518bae233e766e876ac77a0003cd8ed5cd where it was introduced and solve conflicts since then

### ☑️ Checks
- [x] I updated the documentation, including readmes and wikis. If this is a breaking change, update [UPGRADING.md](../UPGRADING.md) to inform users how to proceed. If no updates are necessary, indicate so.
- [x] Tested with dark mode.
- [x] Tested with API 23.

### :test_tube: How can I test this?
- [x] Reviewed by Mistica design team (there isn't anything to review by them)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,11 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        // Remember to update "ui_tooling_preview_version" when updating "compose_bom_version"
-        // according to https://mvnrepository.com/artifact/androidx.compose/compose-bom/
-        // For more info see https://github.com/Telefonica/mistica-android/pull/230
-        compose_bom_version = "2023.01.00"
-        ui_tooling_preview_version = "1.3.3"
+        compose_version = '1.3.1'
```
Comment (yamal-alm) [inline build.gradle:4]:
```suggestion
        compose_version = '1.3.3'
```

**Review by dpastor — APPROVED** [review]
👍  Thanks! :)

**Review by haynlo — APPROVED** [review]
👍🏼

**Review by yamal-alm — COMMENTED** [review]
thanks!

**Pull request merged**